pwiz_tools/Skyline/CommandLine.cs

```diff
@@ -2284,7 +2284,6 @@ private bool AssociateProteins(CommandArgs commandArgs)
                     commandArgs.AssociateProteinsMinPeptidesPerProtein.GetValueOrDefault(),
                     progressMonitor);
                 Settings.Default.LastProteinAssociationFastaFilepath = fastaPath;
-                Settings.Default.Save();

```

Comment:
Shouldn't this also be conditional on commandArgs.SaveSettings?

Reply:
I was assuming that it would be taken care of by the "SaveSettings" call in "CommandLine.RunInner" but I will double check and maybe also write a unit test to verify that.